source/controller/src/map_ctrl_main.c

```diff
@@ -39,6 +39,11 @@
 #include "map_timer_handler.h"
 #include "map_retry_handler.h"
 #include "map_airdata.h"
+#include "ssp_internal.h"
+
+#ifdef INCLUDE_BREAKPAD
+#include "breakpad_wrapper.h"
+#endif
 
```

Comment:
Please add Breakpad support, after rdk-easymesh-controller.bb recipe file is merged.